### DISCUSSION TITLE
policy

### DISCUSSION THREAD

**Discussion by xxdyu** [rabbitmq/rabbitmq-server#2445] [Other] [answered]
![image](https://user-images.githubusercontent.com/32918557/93035216-fd9d4780-f66e-11ea-98f8-45d5ccda0d0b.png)

**Comment by lukebakken** [comment, marked as answer]
The regular expression `*app*` is not valid, since the first star operator `*` does not repeat anything.

My guess is you want to use `.*app.*`.
